index.php

```diff
@@ -146,6 +146,19 @@
 							'classes' => [ 'form-repos-field' ],
 						]
 					),
+					new OOUI\FieldLayout(
+						new OOUI\CheckboxInputWidget( [
+							'name' => 'proxy',
+							'value' => 1,
+							'selected' => false
+						] ),
+						[
+							'label' => 'Proxy articles from en.wikipedia.org',
+							'help' => 'Any content not locally to the wiki will be pulled from production wiki	',
```

Comment:
content->articles
locally->local

or:
"Any articles which don't exist on the wiki will be fetched from English Wikipedia"

Reply:
(done)